include/cvc5/cvc5_proof_rule.h

```diff
@@ -1373,6 +1373,29 @@ enum ENUM(ProofRule)
    * \endverbatim
    */
   EVALUE(SETS_EXT),
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Sets -- Sets filter up**
+   *
+   * .. math::
+   *
+   *   \inferrule{\mathit{set.member}(x,a)\mid P}
+   *   {\mathit{set.member}(x, \mathit{set.filter}(P, a)) = P(x)}
+   *
+   * \endverbatim
+   */
+  EVALUE(SETS_FILTER_UP),
+  /**
+   * \verbatim embed:rst:leading-asterisk
+   * **Sets -- Sets filter down**
+   *
+   * .. math::
+   *
+   *   \inferrule{\mathit{set.member}(x,\mathit{set.filter}(P, a))\mid -}
```

Comment:
Is this `\mid -` required? Can't you have a rule with a single premise?

Reply:
Mid separates premises from "arguments", which this rule has none.